File: CS/DxDataGridExportingWithReportsClientBlazor/DxDataGridExportingWithReportsClientBlazor/Startup.cs
using Microsoft.AspNetCore.Components.Builder;
using Microsoft.Extensions.DependencyInjection;

namespace DxDataGridExportingWithReportsClientBlazor
{
    public class Startup
    {
        public void ConfigureServices(IServiceCollection services) {
            services.AddDevExpressBlazor();
        }

        public void Configure(IComponentsApplicationBuilder app) {
            app.AddComponent<App>("app");
        }
    }
}

File: VB/DxDataGridExportingWithReportsClientBlazor/DxDataGridExportingWithReportsClientBlazor/Startup.vb
Imports Microsoft.AspNetCore.Components.Builder
Imports Microsoft.Extensions.DependencyInjection

Namespace DxDataGridExportingWithReportsClientBlazor
	Public Class Startup
		Public Sub ConfigureServices(ByVal services As IServiceCollection)
			services.AddDevExpressBlazor()
		End Sub

		Public Sub Configure(ByVal app As IComponentsApplicationBuilder)
			app.AddComponent(Of App)("app")
		End Sub
	End Class
End Namespace
